File: pset6/mario.py
import cs50

def main():
    print("Height: ", end="")
    height = cs50.get_int()
    
    while height < 0 or height >= 23:
        #print("Please enter a positive int that is less than 23")
        print("retry: ", end ="")
        height = cs50.get_int()
        
    sharps = 1
    spaces = height-1
    
    
    for i in range(height):
        while spaces >= 0:
            for s in range(spaces):
                print(" ", end="")
            for sh in range(sharps):
                print("#",end="")
            
            print(" ", end ="")
            
            for sh in range(sharps):
                print("#",end="")
            for s in range(spaces):
                print(" ", end="")
                
            #newline    
            print("")
            
            #decrement and increment values
            sharps +=1
            spaces -=1

main()
            
            

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    printf("Height: ");
    
    int height = GetInt();
    while ((height >= 23) ||(height < 0))
    {
        printf("Please enter a positive int that is less than 23\n");
        printf("Retry: ");
        height = GetInt();
    }
    
    int spaces = height - 1;
    int sharps = 2;
    
    for(int i = height; i > 0; i--)
    {
        for (int j = 0; j < spaces; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < sharps; k++)
        {
            printf("#");
        }
        printf("\n");
        spaces--;
        sharps++;
    }
}